2-mandatory/4-restaurant.js

```diff
@@ -19,10 +19,33 @@ const MENU = {
   falafel: 7.25,
 };
 
+function orderBurger(balance) {
+  if (balance - MENU.burger >= 0) {
+    return balance - MENU.burger >= 0
+  }
+}
+
+return balance;
+
+function orderFalafel(balance) {
+  if (balance - MENU.burger >= 0) {
+    return balance - MENU.falafel >= 0
+  }
+}
+
+return balance;
```

Comment:
similar to the above, this should be in the related function :)

Reply:
Fixing the return statement's location would also help with the tests :) https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week1/actions/runs/3861676813/jobs/6582791676